.circleci/scripts/release-create-gh-release

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+if [[ "${CI:-}" != 'true' ]]
+then
+    printf '%s\n' 'CI environment variable must be set to true'
+    exit 1
+fi
+
+if [[ "${CIRCLECI:-}" != 'true' ]]
+then
+    printf '%s\n' 'CIRCLECI environment variable must be set to true'
+    exit 1
+fi
+
+printf '%s\n' 'Installing hub CLI'
+pushd "$(mktemp -d)"
+    curl -sSL 'https://github.com/github/hub/releases/download/v2.11.2/hub-linux-amd64-2.11.2.tgz' | tar xz
+    PATH="$PATH:$PWD/hub-linux-amd64-2.11.2/bin"
+popd
+
+current_commit_msg=$(git show -s --format='%s' HEAD)
+
+if grep --quiet '^Version v' <<< "$current_commit_msg"
+then
+    printf '%s\n' 'Creating GitHub Release'
+    tag="${current_commit_msg/Version /}"
```

Comment:
We should use the 2nd word here as the version, to allow us to always squash+merge, e.g.

```
Version v1.2.3 RC (#foo)
```